### PR TITLE
Pre-release-5.4.3

### PR DESCRIPTION
**Pull request by prestist**
https://github.com/coreos/afterburn/issues/939

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ Packaging changes:
 
 
 
+## Afterburn 5.4.3 (2023-06-05)
+
+Minor changes:
+
+- Cargo update all dependencies to avoid missing data in vendor tarball
+
```
Comment (bgilbert) [inline docs/release-notes.md:22]: One too many trailing newlines.  There should be two between releases.

File: docs/release-notes.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ Packaging changes:
 
 
 
+## Afterburn 5.4.3 (2023-06-05)
+
+Minor changes:
+
+- Cargo update all dependencies to avoid missing data in vendor tarball
```
Comment (bgilbert) [inline docs/release-notes.md]: The last time this happened, in 5.4.1, we said: `Fix missing static archives in vendor tarball`.  They're not static archives in this case; maybe `Fix incomplete package in vendor tarball`?

Re the `cargo update` part, normally dependency updates don't get a release note.  If we were correcting years of missed updates, I'd definitely say it was worth noting, but the last update was b3e25907f7935e6e01a057e8497dd370e1ab419b in January.  So I'd be inclined to skip.

File: docs/release-notes.md (hunk 1 above)
Comment (prestist) [reply docs/release-notes.md:22]: Ugh, sorry I followed the pattern from the (unreleased) section. I should have known better, and double checked.

File: docs/release-notes.md (hunk 2 above)
Comment (prestist) [reply docs/release-notes.md]: Ok, I opted for your suggestion of `Fix missing static archives in vendor tarball`. Thank you for the feedback.

Comment (bgilbert) [reply docs/release-notes.md]: As I said above:

> They're not static archives in this case; maybe `Fix incomplete package in vendor tarball`?

Comment (prestist) [reply docs/release-notes.md]: Sorry I half read the suggestion; no excuse I fixed it.

**Review by bgilbert — APPROVED** [review]

**Pull request merged**